community/index.md

```diff
@@ -120,6 +120,11 @@ See our [events page](/events/).
 Some local Scala user groups are listed on
 [Meetup](https://www.meetup.com/topics/scala/).
 
+ - [London Scala User Group](https://www.meetup.com/london-scala/)
+ - [Warsaw Scala Enthusiasts](https://www.meetup.com/ScalaWAW/)
+ - [Kraków Scala User Group](https://www.meetup.com/Krakow-Scala-User-Group/)
+ - [Wrocław Scala User Group](https://www.meetup.com/wroclaw-scala-user-group/)
```

Comment:
There's also a list on scalanews.net by @softinio at https://www.scalanews.net/Resources/Event_Directory.html

Not sure how up to date that is

Reply:
the other major source of meetup info currently is Scalendar, latest issue https://scalac.io/blog/scalendar-february-2025/

Reply:
Thanks! I added everything from scalanews and this month's Scalendar to the list.